src/ansys/mapdl/core/cndparser.py

```diff
@@ -0,0 +1,248 @@
+#**********************************************************************
+# Importing  used libraries
+#**********************************************************************
+import sys
+import os    
+from enum import Enum
+from decimal import Decimal
+import copy
+import numpy as np
+import xlsxwriter

```

Comment:
We don't require `xlsxwriter`, so one way to work around this is to make it an optional requirement for this library. You'll need to add this in `/requirements/requirements_tests.txt` if you intend on adding unit testing for it as well, along with making it a "lazy import".